PayPal/src/main/java/com/braintreepayments/api/paypal/vaultedit/InternalPayPalVaultEditCallback.kt

```diff
@@ -0,0 +1,16 @@
+package com.braintreepayments.api.paypal.vaultedit
+
+import com.braintreepayments.api.core.ExperimentalBetaApi
+
+/**
+ * Callback for receiving result for edit internally.
+ */
+@ExperimentalBetaApi
+internal fun interface InternalPayPalVaultEditCallback {
```

Comment:
Likely also wont need this class when `InternalPayPalVaultEditAuthRequest` is removed

Reply:
This also matches the current pattern which can be [found here](https://github.com/braintree/braintree_android/blob/main/PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalInternalClientCallback.kt).  I'll rename to more closely match this callback!